src/devices/src/pseudo/init.c

```diff
@@ -0,0 +1,40 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+#include <sys/types.h>
+#include <fcntl.h>
+#include <sys/mman.h>
+#include <unistd.h>
+
+#ifdef __x86_64__
+#define MAGIC_MMIO_SIGNAL_GUEST_BOOT_COMPLETE 0xd0000000
+#endif
+#ifdef __aarch64__
+#define MAGIC_MMIO_SIGNAL_GUEST_BOOT_COMPLETE 0x40001000
+#endif
+
+#define MAGIC_VALUE_SIGNAL_GUEST_BOOT_COMPLETE 123
+
+int main () {
+
+   int fd = open("/dev/mem", (O_RDWR | O_SYNC));
+   int mapped_size = getpagesize();
+
+   char *map_base = mmap(NULL,
+        mapped_size,
+        PROT_WRITE,
+        MAP_SHARED,
+        fd,
+        MAGIC_MMIO_SIGNAL_GUEST_BOOT_COMPLETE);
+
+   char *virt_addr = (char*)map_base;
+
+   *(volatile char*)virt_addr = MAGIC_VALUE_SIGNAL_GUEST_BOOT_COMPLETE;
+
+   const char *init = "/sbin/openrc-init";
```

Comment:
This is `Alpine` specific. Add a comment at the top of the file making it clear that the resulting binary would be `Alpine` specific.

Reply:
Done.